### PR TITLE
Extension of docstrings, fix typos in tests, 

### PR DESCRIPTION
**Pull request by jkoniecznyy**
clean main, delete 2 unnecessary files

### CODE REVIEW

File: graph_theory/helpers.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 def create_random_map(min_length: int = 5, max_length: int = 10) -> list[list[int]]:
     """
-    Creates a random map of 0's and 1's (water/land) with a random number of rows and columns
+    Create a random map of 0's and 1's (water/land) with a random number of rows and columns
```
Comment (daroG) [inline graph_theory/helpers.py]: What is a purpose for changing it?

File: graph_theory/helpers.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def print_map_as_bits(map: list[list[int]]) -> None:
 
 def present_map(map: list[list[int]]) -> None:
     """
-    Prints simple map visualization and count of the map islands
+    Print simple map visualization, count the islands on the map
```
Comment (daroG) [inline graph_theory/helpers.py]: Maybe: Prints simple map visualization, counts the islands on the map

File: graph_theory/map_explorer.py [hunk 3]
```diff
@@ -9,22 +9,42 @@ def __init__(self, map: list[list[int]]):
...
     def _is_fragment_island(self, row_id: int, column_id: int) -> bool:
+        """
+        Check if given coordinates are invalid or visited,
+        if not calls itself to mark the entire island
+        :param row_id: row number
+        :param column_id: column number
+        :return: True if marking is completed
+                 False if coordinates are invalid, visited or there is no land
+        """
+        # 1. Check if coordinates are valid
```
Comment (daroG) [inline graph_theory/map_explorer.py]: What is a purpose for adding that comment (line 33)? It's readable in the code.

File: graph_theory/map_explorer.py [hunk 4]
```diff
@@ -9,22 +9,42 @@ def __init__(self, map: list[list[int]]):
...
             return False
 
+        # 2. Check if coordinates were visited
         if (
             self._map[row_id][column_id] == self.MAP_NOT_ISLAND_MARK
             or self._visited_points[row_id][column_id]
         ):
             return False
 
+        # 3. Mark coordinates as visited
```
Comment (daroG) [inline graph_theory/map_explorer.py]: What is a purpose for adding that comment? It's readable in the code.

File: graph_theory/map_explorer.py [hunk 5]
```diff
@@ -9,22 +9,42 @@ def __init__(self, map: list[list[int]]):
...
+        :param row_id: row number
+        :param column_id: column number
+        :return: True if marking is completed
+                 False if coordinates are invalid, visited or there is no land
+        """
+        # 1. Check if coordinates are valid
         if not self._check_row_and_cols_id(row_id, column_id):
             return False
 
+        # 2. Check if coordinates were visited
```
Comment (daroG) [inline graph_theory/map_explorer.py]: What is a purpose for adding that comment? It's readable in the code.

File: graph_theory/map_explorer.py [hunk 6]
```diff
@@ -9,22 +9,42 @@ def __init__(self, map: list[list[int]]):
...
         if (
             self._map[row_id][column_id] == self.MAP_NOT_ISLAND_MARK
             or self._visited_points[row_id][column_id]
         ):
             return False
 
+        # 3. Mark coordinates as visited
         self._visited_points[row_id][column_id] = True
 
+        # 4. Call itself to go through any land directly connected to the island
```
Comment (daroG) [inline graph_theory/map_explorer.py]: Maybe that part behind should be extracted to another method, so this comment can be removed

File: graph_theory/map_explorer.py [hunk 7]
```diff
@@ -34,19 +54,20 @@ def _is_fragment_island(self, row_id: int, column_id: int) -> bool:
         self._is_fragment_island(row_id - 1, column_id)  # up
         self._is_fragment_island(row_id - 1, column_id - 1)  # up left
 
+        # 5. If marking is completed, return True
```
Comment (daroG) [inline graph_theory/map_explorer.py]: Useless comment, it's readable in the code.

File: graph_theory/helpers.py [hunk 8]
```diff
@@ -17,7 +17,7 @@ def create_random_map(min_length: int = 5, max_length: int = 10) -> list[list[in
 
 def draw_map(map: list[list[int]]) -> None:
     """
-    Draws the map in the console
+    Draw the map in the console
```
Comment (daroG) [inline graph_theory/helpers.py]: See the first comment

File: graph_theory/helpers.py [hunk 9]
```diff
@@ -28,7 +28,7 @@ def draw_map(map: list[list[int]]) -> None:
 
 def print_map_as_bits(map: list[list[int]]) -> None:
     """
-    Prints map as bits, each row in the next line
+    Print map as bits, each row in the next line
```
Comment (daroG) [inline graph_theory/helpers.py]: See the first comment

File: graph_theory/map_explorer.py [hunk 10]
```diff
@@ -34,19 +54,20 @@ def _is_fragment_island(self, row_id: int, column_id: int) -> bool:
...
         return True
 
     def get_islands_count(self) -> int:
         """
-        Returns the islands count
+        Return the islands count
```
Comment (daroG) [inline graph_theory/map_explorer.py]: See the first comment

File: graph_theory/map_explorer.py [hunk 11]
```diff
@@ -34,19 +54,20 @@ def _is_fragment_island(self, row_id: int, column_id: int) -> bool:
...
         :return: islands count
         """
         return self._island_count
 
     def perform_exploring(self) -> None:
         """
-        Goes through the map detecting the land
-        When the land is detected calls the _is_fragment_island and increments the count
+        Go through the map detecting the land
+        When the land is detected call the _is_fragment_island and increment the count
```
Comment (daroG) [inline graph_theory/map_explorer.py]: See the first comment

**Review by daroG — CHANGES_REQUESTED** [review]

File: graph_theory/helpers.py (hunk 1 above)
Comment (jkoniecznyy) [reply graph_theory/helpers.py]: I found a guide for docstrings saying that verbs should be in the infinitive form, but I think it's up to us if we want to keep it in third-person or change to infinitive.
Link: https://pandas.pydata.org/docs/development/contributing_docstring.html#section-1-short-summary

image

File: graph_theory/helpers.py (hunk 2 above)
Comment (jkoniecznyy) [reply graph_theory/helpers.py]: It dosen't make a big difference. I just prefer it this way, but we can do yours. Maybe @tdxa will help us with this dilema ;p

**Review by tdxa — APPROVED** [review]

**Pull request closed without merging**